src/libm/sleefinline_header.h.org

```diff
@@ -5,7 +5,11 @@
 // This file is generated by SLEEF SLEEF_VERSION_SLEEF
 
 #ifndef SLEEF_ALWAYS_INLINE
-#define SLEEF_ALWAYS_INLINE inline
+#if defined (__GNUC__) || defined (__clang__) || defined(__INTEL_COMPILER)
+#define SLEEF_ALWAYS_INLINE inline __attribute__((always_inline))
+#else
+#define SLEEF_ALWAYS_INLINE inline __forceinline
```

Comment:
Since `__forceinline` is MSVC specific I'd prefer to have a separate elif for MSVC and the fallback case to just be `inline`.

Reply:
@blapie fixed.